### PR TITLE
[KOGITO-5797] Allow change topic name for event process springboot addon

### PR DESCRIPTION
**Pull request by fjtirado**
See [JIRA](https://issues.redhat.com/browse/KOGITO-5797) for description of this PR

### CODE REVIEW

File: springboot/addons/messaging/implementation/src/main/java/org/kie/kogito/addon/cloudevents/spring/SpringKafkaCloudEventEmitter.java [hunk 1]
```diff
@@ -42,8 +42,6 @@
 
     @Autowired
     org.springframework.kafka.core.KafkaTemplate<String, String> emitter;
-    @Value(value = "${spring.kafka.bootstrap-servers}")
```
Comment (fjtirado) [inline springboot/addons/messaging/implementation/src/main/java/org/kie/kogito/addon/cloudevents/spring/SpringKafkaCloudEventEmitter.java:45]: This is dead code in springboot addon that Im taking the chance to remove

**Review by ricardozanini — APPROVED** [review]

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
@fjtirado overall looks good, my only comment would be if we could add some test

**Comment by fjtirado** [comment]
jenkins retest this

**Comment by fjtirado** [comment]
@cristianonicolai please take a look at this

**Comment by fjtirado** [comment]
> @fjtirado overall looks good, my only comment would be if we could add some test

I can modify multi boot example in a different pr

**Comment by cristianonicolai** [comment]
@fjtirado we do end-to-end in apps, but I guess you're right we're missing that in kogito-runtimes, we should add it to integration-tests and use `KafkaTestClient` to verify events are published

**Comment by fjtirado** [comment]
> @fjtirado we do end-to-end in apps, but I guess you're right we're missing that in kogito-runtimes, we should add it to integration-tests and use `KafkaTestClient` to verify events are published

Ok, I have created JIRA https://issues.redhat.com/browse/KOGITO-5821 to add integration tests  (It will alter my plannification to this sprint to do that with this one)

**Pull request merged**